dockertest/environment.py

```diff
@@ -47,12 +55,7 @@ def get_selinux_context(path):
     When selinux is enabled, return the context of ``path``
     :param path: Full or relative path to a file or directory
     :return: SELinux context as a string or None if not set
+    :raises IOError: When file does not exist.
```

Comment:
It's nice when ``get_*()`` and ``set_*()`` match behavior closely, but this is a very specific difference in this case.  So I felt it would be nice to at least document it.  Are you afraid this implies other exceptions are not raised?